tests/init-with-example/__snapshots__/init-with-example-with-log.test.js.snap

```diff
@@ -37,12 +37,18 @@ Array [
       ],
     },
   },
+  Object {
+    "log": Array [
+      "ℹ",
+      "OK, continuing with isView: false",
```

Comment:
I think it would be ideal to make this info log message a little more user-friendly.